centinel/experiments/baseline.py

```diff
@@ -101,10 +94,17 @@ def run_file(self, input_file):
                     file_comments.append(row)
                 continue
 
+            # detect the header row and store it
```

Comment:
Please include an example of what such a file should look like because it will make this code easier to read.